Source/WebKit/UIProcess/glib/WebProcessPoolGLib.cpp

```diff
@@ -163,8 +163,9 @@ void WebProcessPool::platformInitializeWebProcess(const WebProcessProxy& process
         parameters.accessibilityBusAddress = m_sandboxEnabled && shouldUseBubblewrap() ? sandboxedAccessibilityBusAddress() : accessibilityBusAddress();
 #endif
 
+    parameters.systemSettings = WebCore::SystemSettingsGLib::singleton().settingsState();
```

Comment:
We probably need a SystemSettingsManager or SystemSettingsProvider or something like that with implementations for GTK and WPE to be used in the UI process.